lldb/include/lldb/Target/Thread.h

```diff
@@ -128,6 +128,9 @@ class Thread : public std::enable_shared_from_this<Thread>,
         register_backup_sp; // You need to restore the registers, of course...
     uint32_t current_inlined_depth;
     lldb::addr_t current_inlined_pc;
+    lldb::addr_t stopped_at_unexecuted_bp; // Set to the address of a breakpoint
```

Comment:
This is the stored version of the Thread ivar with roughly the same name.  It's better documented there, maybe here you can just refer to that ivar for docs?